lib/initializers/events-responder.js

```diff
@@ -1,31 +1,51 @@
-const formatTime = require('../helpers/format-time')
-const formatRelativeTime = require('../helpers/format-relative-time')
-const matrixToString = require('../helpers/matrix-to-string')
-const getEventsFetcher = require('../fetchers/events-fetcher')
+// Description:
+//   Get upcoming event information.
+//
+// Dependencies:
+//   "@firebase/firestore"
+//   "moment"
+//
+// Configuration:
+//   N/A
+//
+// Commands:
+//   upcoming events - Get a table of upcoming events by date and group.
+//   next <group> event - Get the details of the next event for the given group.
+//
+// Notes:
+//   N/A
+//
+// Author:
+//   egillespie
+
+const formatTime = require('../helpers/format-time');
+const formatRelativeTime = require('../helpers/format-relative-time');
+const matrixToString = require('../helpers/matrix-to-string');
+const getEventsFetcher = require('../fetchers/events-fetcher');
 
 module.exports = robot => {
   // When you hear someone talking/asking about an upcoming event...
   robot.hear(/(?:next|upcoming) (.*) (?:meetup|event)/i, message => {
     // Get the type of group being searched for
-    const group = message.match[1]
+    const group = message.match[1];
```

Comment:
Do you mind removing the semicolons? I'd like to move these projects all toward a common linting configuration and I think at a minimum this would follow the [Standard JavaScript linting rules](https://standardjs.com/rules.html#semicolons), which declares no semicolons.

I'll create an issue to introduce actual linting tools later.

Reply:
Not at all! I was wondering while I was working on this whether there was a style or linter I ought to be using. Thanks for the pointer!

Reply:
I've opened the relevant issue, #44 . Please let me know if I missed anything.